apps/foundation-demo/src/app/design/atoms/form/select/select.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: '[daff-select]',
+  host: {'class': 'daff-select'},
+  template: '<ng-content></ng-content>',
+  styleUrls: ['./select.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class DaffSelectComponent {
```

Comment:
doesn't look like this component does anything

Reply:
Going to remove it and move it to a new PR for `form controls`